docs/GettingStartedDocs/VisualStudioWindows.md

```diff
@@ -0,0 +1,93 @@
+# Using Visual Studio to Develop Enclave Applications for Windows
+
+This walkthrough assumes that you are using Visual Studio on a Windows development
+machine, and want to develop enclaves for Windows.  If you instead want to use Visual
+Studio Code, see the
+[VS Code instructions](https://github.com/openenclave/openenclave/blob/master/devex/vscode-extension/README.md).
+
+## Prerequisites
+
+You will need the following:
+
+- A development machine with support for SGX1 or SGX1 with Flexible Launch Control (FLC) (see [instructions for determining the SGX support level](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/SGXSupportLevel.md) if needed).
+- [Visual Studio Preview](https://visualstudio.microsoft.com/vs/preview/)
```

Comment:
This is already publicly available: https://devblogs.microsoft.com/visualstudio/visual-studio-2019-version-16-5/
There is special mention of Open Enclave debugging support.
Kindly update the placeholder link `aka.ms/OpenEnclaveVisualStudio` to point to the right location.

Reply:
Updated file here.  Will update aka.ms link once the update is released into the VS marketplace.